doc/architecture/decisions/0005-data-retention-policy.md

```diff
@@ -0,0 +1,111 @@
+
+```mermaid
+gantt
+    title Current Policy
+    dateFormat  YYYY-MM-DD
+
+    section User Journey
+    User completes application: uj1, 2022-01-01, 28d
+
+    section User Data
+    Lowcal Session: 2022-01-01, 35d
+    Uploaded files: 2022-01-14, 2022-12-31
+    Session backups: after uj1, 2022-12-31
+
+    section Audit Logs
+    Submission audit logs: after uj1, 2022-12-31
+    Reconciliation audit logs: 2022-01-14, 2022-12-31
+
+    section Event Metadata
+    Submission Event Metadata: after uj1, 2022-12-31
+    Reconciliation Event Metadata: 2022-01-14, 2022-12-31
+    
+    section Feedback
+    FeedbackFish (last 250 records, rolling): 2022-01-14, 2022-12-31
+    Feedback Learning Log (anonymised): 2022-01-14, 2022-12-31
+```
+
+```mermaid
+gantt
+    title Proposed Policy
+    dateFormat  YYYY-MM-DD
+
+    section User Journey
+    User completes application: uj1, 2022-01-01, 28d
+
+    section User Data
+    Lowcal Session: 2022-01-01, 210d
+    Lowcal Session (Sanitised): 2022-12-31
+    Uploaded files: 2022-01-14, 182d
+    Session backups: after uj1, 182d
+    Session backups (Sanitised): 2022-12-31
+
+    section Audit Logs
+    Submission audit logs: after uj1, 182d
+    Submission audit logs (Sanitised): 2022-12-31
+    Reconciliation audit logs: 2022-01-14, 182d
+
+    section Event Metadata
+    Submission Event Metadata: after uj1, 182d
+    Reconciliation Event Metadata: 2022-01-14, 182d
+
+    section Feedback
+    FeedbackFish (last 250 records, rolling): 2022-01-14, 2022-12-31
+    Feedback Learning Log (anonymised): 2022-01-14, 2022-12-31
+```
+
+Here are the broad catergory of changes - 
+
+|| Current Retention Period | Proposed Retention Period |
+|---|---|---|
+| **User Data**| Lowcal Sessions - 7 days, then deleted. Session Backups - Indefinite | 6 months from submission date, then sanitised and held indefinitely |
+| **Uploaded Files**| Indefinite | 6 months from upload date |
+| **Audit Logs** | Indefinite | Submission Logs - 6 months from submission date, then sanitised and held indefinitely. <br /> Reconciliation Logs - 6 months from submission date, then deleted  |
+| **Event Metadata** | Indefinite | 6 months from creation date, then deleted | 
+| **Feedback** | Indefinite | Indefinite
+
+
+### Timetable
+The new policy will be implemented once this proposal has been accepted. This means that records over 6 months old at the point the feature is merged to production will be sanitised or deleted.
+
+An automated scheduled task will run each night to sanitise and delete records which meet the agreed criteria.
+
+### What does "Sanitation" mean?
+Currently, we would simply remove / delete and user data (e.g. email, breadcrumbs, passport) whilst retaining the remainder of the metadata associated with a record (such as `id`, `created_at`). This would allow to still have enough information to resconstruct user journeys if required to do so for auditing purposes.
```

Comment:
Is this true though? If we delete breadcrumbs, can we still reconstruct the user journey?

Also, small typo on "resconstruct"

Reply:
Thanks for this - I've not clarified my meaning here. What I'm referring to is "high level" user journeys (e.g. when was an application started, payed for, and submitted) as opposed to the granular question by question journey.